administrator/language/de-DE/com_contact.ini

```
@@ -37,6 +37,7 @@ COM_CONTACT_FIELD_CONFIG_CATEGORIES_DESC="Diese Einstellungen gelten für die Ko
 COM_CONTACT_FIELD_CONFIG_CATEGORY_DESC="Diese Einstellungen gelten für eine einzelne Kontaktkategorie, wenn sie nicht in einem eigenen Menülink überschrieben werden."
 COM_CONTACT_FIELD_CONFIG_CONTACT_FORM="Formular"
 COM_CONTACT_FIELD_CONFIG_COUNTRY_LABEL="Land"
+COM_CONTACT_FIELD_CONFIG_CUSTOM_REPLY_DESC="Verhindert, dass Joomla die E-Mail sendet, sodass Plugins die Integration mit anderen Systemen übernehmen können."
```

```suggestion
COM_CONTACT_FIELD_CONFIG_CUSTOM_REPLY_DESC="Verhindert, dass die E-Mail durch Joomla versendet wird. Dies erlaubt Plugins die Integration mit anderen Systemen zur Übertragung der Daten des Kontaktformulars"
```

Bin immer noch nicht ganz glücklich mit der Formulierung aber für mich ist es jetzt zumindest etwas klarer was gemeint ist.

---

Aber Kontaktformular ist da fehl am Platz. Ich schaue mir das später nochmals an.